samples/bluetooth/peripheral_esp/src/main.c

```diff
@@ -109,7 +155,7 @@ static struct temperature_sensor sensor_2 = {
 		.lower_limit = -1000,
 		.upper_limit = 5000,
 		.condition = ESS_VALUE_CHANGED,
-		.meas.sampling_func = 0x00,
+		.meas.sampling_func = ESS_UNSPECIFIED,
 		.meas.meas_period = 0x01,
 		.meas.update_interval = SENSOR_2_UPDATE_IVAL,
 		.meas.application = 0x1b,
```

Comment:
Missing update for `.meas.application = 0x1b,`?

Reply:
I made the update for the missing 0x1b.